game/configuration.cc

```diff
@@ -352,6 +392,21 @@ void readConfigXML(fs::path const& file, int mode) {
 	}
 }
 
+int PaHostApiNameToHostApiTypeId (const std::string& name) {
+	if (name == "Auto") return 1337;
+	if (name == "Windows DirectSound") return 1;
+	if (name == "MME") return 2;			
+	if (name == "ASIO") return 3;
+	if (name == "Core Audio" || name == "CoreAudio") return 5;
+	if (name == "OSS") return 7; // Not an error, stupid PortAudio.
+	if (name == "ALSA") return 8;
```

Comment:
when I set the config to "auto" pulseaudio is detected as audio device, but it's not shown in the config option, is this correct?